### DISCUSSION TITLE
Changing enrollment day in pokemon SV

### DISCUSSION THREAD

**Discussion by chensnowps4** [kwsch/PKHeX#3713] [Q&A] [answered]
How may I edit my enrollment date in pokemon SV?
Thank you very much

**Comment by Lusamine** [comment]
You can see this issue for an explanation of how to edit that block: https://github.com/kwsch/PKHeX/issues/3800

**Comment by foohyfooh** [comment, marked as answer]
I have made a plugin which should allow you to do this. Go to the releases page on [https://github.com/foohyfooh/PKHeXPluginPile](https://github.com/foohyfooh/PKHeXPluginPile) and you can download it.

**Reply by chensnowps4** [reply]
Thank you so much!!! you save my save file!

**Comment by kwsch** [comment]
Enrollment date can now be edited via the GUI via the Trainer Editor.

**Reply by chensnowps4** [reply]
Thank you so much!!!!!!!
